src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -0,0 +1,2394 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+
+namespace Microsoft.CodeAnalysis.Semantics
```

Comment:
are we checking in generated .cs files?  i thought the pattern was that an initial compilation step just generated them for you.